flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/BinaryWriter.java

```diff
@@ -94,8 +97,11 @@ static void write(BinaryWriter writer, int pos,
 			case INTERVAL_YEAR_MONTH:
 				writer.writeInt(pos, (int) o);
 				break;
-			case BIGINT:
 			case TIMESTAMP_WITHOUT_TIME_ZONE:
+				TimestampType timestampType = (TimestampType) type;
```

Comment:
You can not modify static utils right now? This will lead to tests failure.

Reply:
Sure